src/ray/core_worker/core_worker.cc

```diff
@@ -2711,45 +2711,54 @@ Status CoreWorker::CreateActor(const RayFunction &function,
       local_mode_named_actor_registry_.emplace(actor_name, actor_id);
     }
     ExecuteTaskLocalMode(task_spec);
+    return Status::OK();
+  }
+
+  task_manager_->AddPendingTask(
+      rpc_address_,
+      task_spec,
+      CurrentCallSite(),
+      // Actor creation task retry happens through the gcs, so the task manager only
```

Comment:
this should also describe why we have the `!task_spec.IsDetachedActor()` condition